MIPs/mip-x-evmmultichainapi.md

```diff
@@ -0,0 +1,168 @@
+---
+mip: x
+title: Multichain API for Ethereum
+status: Draft
+stability: n/a
+discussions-to: https://github.com/MetaMask/metamask-improvement-proposals/discussions
+author(s): Alex Donesky (@adonesky1), Jiexi Luan (@jiexi), Vandan Parikh(@vandan) 
+type: Maintainer
+created: 2024-10-28
+---
+
+## Summary
+This proposal supplements [MIP-x](./mip-x-multichainapi.md) with additional specifications for Ethereum-compatible networks.
+
+## Motivation
+Developers building against Ethereum-compatible networks stand to benefit the most from the simplified developer experience associated with the Multichain API. 
+
+The Multichain API and associated CAIP standards promise to enable the following benefits for these developers:
+- Elimination of excessive error handling involved with chain-switching across EVM networks
+- Ability to use interface negotiation to adopt innovative wallet features, while gracefully degrading for wallets that may not yet support them
+- Ability to simultaneously interact with multiple networks
+- Be notified of updates across multiple networks
+
+## Usage Example
+### Request
+An example structure for a JSON-RPC request that an application would send to request authorization(s) from MetaMask:
+
+```js
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_createSession",
+  "params": {
+    "optionalScopes": {
+      "wallet:eip155": {
+        "methods": ["wallet_addEthereumChain"],
+      },
+      "eip155:1": {
+        "methods": ["eth_sendTransaction","eth_call","eth_getBalance","eth_blockNumber","eth_getTransactionCount","wallet_watchAsset","eth_subscribe"],
+        "notifications": ["message"],
+      },
+      "eip155:59144": {
+        "methods": ["eth_sendTransaction","eth_call","eth_getBalance","eth_blockNumber","eth_getTransactionCount","wallet_watchAsset","eth_subscribe"],
+        "notifications": ["message"],
+      },
+    },
+    "scopedProperties": {
+      "eip155:1": {
+        "rpcEndpoints": [{ 
+          "chainName": "Ethereum (Infura)",
+          "rpcUrls": ["https://mainnet.infura.io"],
+          "nativeCurrency": {
+              "name": "ETH",
+              "symbol": "ETH",
+              "decimals": 18,
+          },
+          "iconURLs": ["https://example.com/ethereum.svg"] 
+        }],  
+      },
+      "eip155:59144": {
+        "rpcEndpoints": [{ 
+          "chainName": "Linea (Infura)",
+          "rpcUrls": ["https://rpc.linea.build"],
+          "nativeCurrency": {
+              "name": "ETH",
+              "symbol": "ETH",
+              "decimals": 18,
+          },
+          "iconURLs": ["https://example.com/linea.svg"] 
+        }],
+      },
+    },
+  },
+}
+```
+
+### Response
+An example structure for the corresponding JSON-RPC response that an application would receive from the wallet:
+
+```js
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "sessionScopes": { 
+      "eip155:wallet": {
+          "methods": ["wallet_addEthereumChain"],
```

Comment:
@adonesky1 we're just going to return all methods and notifications to start. worth calling that out in this example? Maybe with `...`?